### PR TITLE
Release notes for `1.36.x - Release #2`

### LINKED ISSUE #21642: Release notes for `1.36.x - Release #2`
### Release Notes:

* Upgrade from Chromium 99.0.4844.45 to Chromium 99.0.4844.51. #21370
* Crash on wallet creation with fingerprint unlock setup on some devices #21153
* Investigate and fix Arbitrum transaction failures #21084
* [Follow up to #20967] Custom network is still removed when editing chainID when its not set as default #21046
* Retire stack layout UI for tabs overview mode #21005
* Buy & Swap screen should be closed when an unsupported network is selected #20977
* Button text at the bottom of custom network form is cut off #20962
* Stop Crypto Wallet extension if default wallet is set to a different wallet #20905
* Problems casting votes on Snapshot w/ Ledger #20864
* Temporarily disable EIP-1559 on Polygon and Avalanche #20652
* Incorrect Asset icon on Brave Wallet (Crypto) #20649
* After connecting to https://lostpoets.xyz/ personalSign request does not work. #20600
* Wallet backup sends user to verify recovery phrase page the first time after reset #20532
* Cursor resets to starting position when Show recovery phrase is selected #20465
* Link to connected site list in the panel #20380
* Password and recovery phrase values are being persisted after restore action #20370
* Support EVM compatible chains #20337
* Add an extra validation of ETH address #20132
* Biometric unlock not coming up when autolock happens within wallet #20036
* Same token should not be listed for both Swap from and to #20005
* Partition WebSockets Limits to prevent side channels #19990
* Fingerprint unlock is only shown when you launch wallet #19973
* Selecting unapproved transaction doesn't bring up the approval popup #19969
* Add ability to scroll Tab groups bar icons on swipe #19902
* Theme related issues on Android Wallet #19827
* Change the description for the setting Automatically sends crash reports to Brave #19822
* Add Ethereum chain network dialog should list number of decimals too #19749
* "Hide Brave Rewards button" stands out from the other toggles #19695
* Add an option to disable autocomplete completely #19647
* Add restore from brave crypto wallets checkbox for 24-word mnemonic #18459
* Add chain validation to wallet_addEthereumChain api call #17640
* Get rid of protobufs in P3A pings #15967
* Disable default peeking card #21477

### PR DESCRIPTION
**Pull request by kjozwiak**
Resolves #21642

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:

N/A

### CODE REVIEW

File: CHANGELOG_ANDROID.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## [1.36.111](https://github.com/brave/brave-browser/releases/tag/v1.36.111)
+
+ - Added EVM support for compatible chains under Brave Wallet. ([#20337](https://github.com/brave/brave-browser/issues/20337))
+ - Added ability to restore Brave Crypto wallets within Brave Wallet using legacy 24-word mnemonic. ([#18459](https://github.com/brave/brave-browser/issues/18459))
+ - Added chain validation to wallet_addEthereumChain API call for Brave Wallet. ([#17640](https://github.com/brave/brave-browser/issues/17640))
```
Comment (SergeyZhukovsky) [inline CHANGELOG_ANDROID.md:7]: we don't have that feature on Android

File: CHANGELOG_ANDROID.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+ - Added "Connected sites" button to the "More" menu for Brave Wallet. ([#20380](https://github.com/brave/brave-browser/issues/20380))
+ - Added "Decimals of precision" to the add network panel for Brave Wallet. ([#19749](https://github.com/brave/brave-browser/issues/19749))
```
Comment (SergeyZhukovsky) [inline CHANGELOG_ANDROID.md:9]: the fix isn't done on Android

File: CHANGELOG_ANDROID.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## [1.36.111](https://github.com/brave/brave-browser/releases/tag/v1.36.111)
+
+ - Added EVM support for compatible chains under Brave Wallet. ([#20337](https://github.com/brave/brave-browser/issues/20337))
+ - Added ability to restore Brave Crypto wallets within Brave Wallet using legacy 24-word mnemonic. ([#18459](https://github.com/brave/brave-browser/issues/18459))
+ - Added chain validation to wallet_addEthereumChain API call for Brave Wallet. ([#17640](https://github.com/brave/brave-browser/issues/17640))
+ - Added "Connected sites" button to the "More" menu for Brave Wallet. ([#20380](https://github.com/brave/brave-browser/issues/20380))
```
[the remark this answers is not in the document]
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: Removed from notes 👍

File: CHANGELOG_ANDROID.md (hunk 1 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: Removed from notes 👍

File: CHANGELOG_ANDROID.md (hunk 2 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: Removed from notes 👍

**Review by srirambv — APPROVED** [review]
++

**Pull request merged**